server/embed/config.go

```diff
@@ -178,6 +179,10 @@ type Config struct {
 	MaxTxnOps           uint   `json:"max-txn-ops"`
 	MaxRequestBytes     uint   `json:"max-request-bytes"`
 
+	// This is the time duration after which a warning is generated if applying request
+	// takes more time than this value.
+	ExperimentalWarningApplyDuration time.Duration `json:"experimental-warning-apply-duration"`
```

Comment:
NIT: could you move this flag to the code block around line 290? It looks like all the experimental flags are grouped there.

Reply:
@jingyih Done